packages/commonwealth/client/scripts/hooks/useWallets.tsx

```diff
@@ -104,6 +105,13 @@ const useWallets = (walletProps: IuseWalletProps) => {
   });
 
   useEffect(() => {
+    if (app.config.evmTestEnv === 'e2e-test') {
+      window['ethereum'] = new MockMetaMaskProvider(
+        'https://eth-mainnet.alchemyapi.io/v2/BCNLWCaGqaXwCDHlZymPy3HpjXSxK7j_',
```

Comment:
Permissions or RPC should be set up here before merge. CI works locally w/ approved RPC

Reply:
What github config do I need to set up? app.config is the response from the /status route -- I was expecting us to use the webpack DefinePlugin to use an env var, but this is a fine approach too.

Reply:
I actually think DefinePlugin is preferable because then the unused/testing codepaths will be pruned during codegen.

Reply:
@jnaviask Any reference in our code of how to do this with definePlugin? No config should be required with the current set up as I'm setting the var in CI

Reply:
It's not clear to me how app.config.evmTestEnv gets populated...

And look up `COSMOS_GOV_V1` for an example of DefinePlugin.

Reply:
@jnaviask app.config.evmTestEnv is an existing env var setup to support FE use of testnets which is populated via status. Env var itself is ETH_RPC

Does the most recent commit implement definePlugin correctly?

Reply:
It added the definePlugin but need to make sure you're referencing the `process.env.ETH_RPC` in the code rather than `app.config`.

Reply:
@jnaviask Ok, does ETH_RPC need to be set in a way other than exporting to test this? Undefined at process.env after switching to that?

Reply:
@jnaviask New login test passing with the updated RPC(thanks for the alchemy access), so this should be good to go.

The failing test is un-related to this change in a different file, should this be addressed as part of this PR?